src/main/java/jp/ac/keio/bio/fun/xitosbml/image/SpatialImage.java

```diff
@@ -322,15 +336,28 @@ public void setAdjacentsList(ArrayList<ArrayList<String>> adjacentsList) {
 	 * @param path the path to the directory
 	 * @param name the name of TIFF file
 	 */
-	public void saveAsImage(String path, String name){
+	public void saveAsImage(String path, String name) {
 		FileSaver fs = new FileSaver(img);
-		if(name == null) return;
-		if(depth > 1)
+		if (name == null)
+			return;
+		if (depth > 1)
 			fs.saveAsTiffStack(path + "/" + name + ".tiff");
 		else
 			fs.saveAsTiff(path + "/" + name + ".tiff");
 	}
 
+	/**
+	 * Save image as TIFF file --> for the CUI version
+	 * 
+	 * @param path2
+	 */
+	public void saveAsImageCui(String path2) {
```

Comment:
2 in this argument does not mean anything so rename to `path`?

Reply:
Sure. I kept the names just to keep track of what parameters I was passing to the functions.

Reply:
Done. Please refer to https://github.com/Medha-B/XitoSBML/issues/5.